### DISCUSSION TITLE
MVP Criteria

### DISCUSSION THREAD

**Discussion by jonwalz** [jonwalz/crypto-data#19] [General]
Let's use this discussion to define the MVP criteria. Which features should be our first goal?
For example, I think we should focus on a single channel of alerts such as email or sms notification then we can expand once we have it working full cycle.

MVP criteria off the top of my head:
- User signup
- Basic watchlist
- Defined set of alert options
- Sends alert via email (or sms?)

Defining alert options will be the biggest lift and uncover the most necessary features to build. Let's look for other alert style products (like stock exchange alerts) to get ideas.

**Comment by jonfreihofer** [comment]
These ideas work for me! To expand on defined set of alert options, were you thinking % change, etc?

**Reply by jonwalz** [reply]
Yeah, checkout this site (the one made by my coworker) https://cryptocurrencyalerting.com/

Lot's of different criteria for triggering alerts.

**Comment by jonwalz** [comment]
A couple sites to take UI/UX ideas from:
- https://coinmarketcap.com/
- https://www.coingecko.com/en/coins/bitcoin
